app/views/report/_column_lists.html.haml

```diff
@@ -44,31 +52,23 @@
           :size     => 10,
           :id       => "selected_fields")
       %td{:width => "30", :valign => "middle"}
```

Comment:
@romanblanco You can follow what I did here: https://github.com/ManageIQ/manageiq/pull/6394/files